### ISSUE TITLE
Websocket returns error through tunnel

### ISSUE THREAD

**Issue by jsmestad** [agrinman/tunnelto#31]
I had to switch back to ngrok for the time being as I cannot connect a websocket through the tunnel. 

When trying to connect, the browser get's stuck throwing `Invalid frame header` while trying to connect. This is with a hello world Phoenix app that relies on LiveView if you need something to recreate it.

Let me know if I can help any more in tracking down the cause.

**Comment by hirschenberger** [comment]
Same issue here. Very sad.

**Comment by agrinman** [comment]
Thanks for the bug report, will look into this

**Comment by gzimh** [comment]
I don't think this is realted to tunnelto. Inavlid frame header is usually a result of SSL mismatch. 
The client request should be secured (e.g socket.io):
```
const socket = io.connect('https://<subdomian>.tunnelto.dev', {secure: true});
```

or, just use the `http` version of the generated tunnel:
```
const socket = io.connect('http://<subdomian>.tunnelto.dev');
```

**Comment by agrinman** [comment]
So yes it turns out this is something tunnelto needs to handle as it was previously not processing the WebSocket upgrade. #42 add supports for this and will be merged shortly...

**Comment by agrinman** [comment]
Fixed it 0.1.16!
